src/transactions/utils/validation.js

```diff
@@ -61,5 +61,17 @@ export const validateAddress = address => {
 			'Address format does not match requirements. Expected "L" at the end.',
 		);
 	}
+
+	const addressAsBignum = bignum(address.slice(0, -1));
+	const biggestAddress = bignum
+		.fromBuffer(Buffer.from([255, 255, 255, 255, 255, 255, 255, 255]))
+		.plus(1);
```

Comment:
Why `plus(1)`?

Reply:
Shouldnt the biggest address still be a possible address?